test/MockContract.js

```diff
@@ -447,7 +457,7 @@ contract('MockContract', function(accounts) {
       // No mock set
       const request = complex.acceptUintReturnString.request([42])
       response = await utils.getRPCResult(request.params);
-      assert.equal(response.result, "0x")
+      assert.equal(response.result, ZERO)
```

Comment:
just out of interest, if `0x0000000000000000000000000000000000000000000000000000000000000000` is decoded as a string it should be an empty string, right? :D

Reply:
This would explain why functions returning `bytes` gives `null` as opposed to the expected "0x0" as @DmitryBespalov recently pointed out.